common/components/Forms/AddServiceForm/index.tsx

```diff
@@ -18,21 +18,20 @@ interface Props {
 const AddServiceForm: React.FC<Props> = ({ form, edit, currentService }) => {
   const { MonthPicker } = DatePicker
   const initialValues = useInitialValues(currentService)
+  console.log(currentService)
 
   useEffect(() => {
-    form.setFieldsValue({
-      date: dayjs(),
-    })
-  }, [])
+    if (currentService) {
+      form.setFieldsValue(initialValues)
+    }
+  }, [currentService, form, initialValues])
```

Comment:
Can we siplify that hook? For example, move all logic from `useInitialValues()` here and handle internally. Please, research is so it look like example below. `useInitialValues` hook is 100% non-reusable, so we can get rid of it:

```
useEffect(() => {
    if (!currentService) {
        return
    }
    
    const initialValues = {
        // TODO: create initital values as it does in `useInititalValues`...
    }

    form.setFieldsValue(initialValues)
}, [form, currentService])
```